lib/diagnostic.rb

```diff
@@ -20,15 +20,24 @@ def example
 # used, and how it differs from Test Driven Development.
 
 # your answer here
+# BDD is an approach that relies on testing to identify the areas of code
+# that are not working properly.
+# It differs from Test Drive Development in that it relies on the flow of
+# data/errors to point the developer to the next step instead of relying on the
+# developer to locate said issue.
 
-#
 # Question 2
 #
 # Create a request spec for our Examples API that ensures 'GET /examples'
 # responds successfully and lists all examples.
 
 RSpec.describe 'Examples API' do
-  # your test(s) here
+  describe 'GET /examples'
+  it 'lists all examples' do
+    get '/examples'
+
+    expect(response).to be_success
+  end
```

Comment:
Want to make sure we also check that the examples coming back from the successful response are what we want with one or both of:

```
expect(examples_response.length).to eq(examples.count)
expect(examples_response['examples'].first['title']).to eq(example[:title])
```